rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -127,16 +117,15 @@ def __next__(self):
                     self.output = np.empty(self.dimensions, dtype=self.dtype)
```

Comment:
@LakshmiKumar23 Can we use the same dlpack -> tensor conversion for Host backend as well? It would return tf.Tensors on CPU device
`output = tf.experimental.dlpack.from_dlpack(self.output_tensor_list[i].__dlpack__(self.device_id))`
Here the device ID is used for creating DLDevice but device type is set as kDLCPU for Host backend